src/radiation/source_terms_single_group.hpp

```diff
@@ -47,9 +47,12 @@ void RadSystem<problem_t>::AddSourceTermsSingleGroup(array_t &consVar, arraycons
 		const double Erad0 = consPrev(i, j, k, radEnergy_index);
 		AMREX_ASSERT(Erad0 > 0.0);
 
+		const double cscale = c / chat;
```

Comment:
The changes in this file look unrelated to particles. Can you put them in a separate PR?

Reply:
This is used later to convert radEnergySource to Src, as I explained in my other comments.